sig-and-wg/sig-core/proposals/installer.md

```diff
@@ -0,0 +1,44 @@
+# Kyma Installer - dynamic installation 
+
+Created on 2018-08-23 by Krysian Cieślik (@crabtree) and Jakub Kabza (@jakkab).
+This document is a part of [Kyma modularization proposal](./modularization.md).
+
+## Status
+
+Submitted for approval on 2018-08-23.
+
+## Motivation
+
+Kyma installation is a complex, time-consuming process, involving multiple components. The procedure is followed every day by a large number of developers who test their features, and thus should easily handle different scenarios without being modified. This is where the dynamic installation comes in handy, unifying, simplifying and automating the process.
+
+## Goal
+
+This proposal aims to enhance Developer Experience by:
+
+- extending installation customizability; 
+- reducing the number of different installation methods;
+- avoiding redundant modifications to the Installer component.
+
+## Current solution
+
+Kyma installation:
+
+- consists of hard-coded steps;
```

Comment:
No punctuation mark at the end of each line, since these bullet points are not full sentences.